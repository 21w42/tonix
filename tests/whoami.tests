basename t
basename --help
help basename
man basename
basename /usr/share
basename /etc/motd
quit

