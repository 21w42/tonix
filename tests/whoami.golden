Welcome to Tonix.
Type "help" to get a list of commands.
"man <COMMAND>" or "help <COMMAND>" sometimes might be helpful.
Some options for certain commands work as well.
Feel free to navigate a pre-made file system using intuitive commands.
Path resolution does not work yet, one step at a time please.
Your feedback is highly appreciated!
Have fun :)
$ t

$ Usage: 	basename NAME

Print NAME with any leading directory components removed.
	-a		support multiple arguments and treat each as a NAME
	-s		remove a trailing SUFFIX; implies -a
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ Usage: 	basename NAME

Print NAME with any leading directory components removed.
	-a		support multiple arguments and treat each as a NAME
	-s		remove a trailing SUFFIX; implies -a
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ basename		User Commands
NAME
	basename - strip directory and suffix from filenames
SYNOPSIS
	basename NAME
DESCRIPTION
	Print NAME with any leading directory components removed.
	-a		support multiple arguments and treat each as a NAME
	-s		remove a trailing SUFFIX; implies -a
	-z		end each output line with NUL, not newline
	--help		display this help and exit
	--version	output version information and exit

$ share

$ motd

$ Bye.
